bin/update-ip.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Gets the ip address of the server container and sets it in place in
+# your /etc/hosts file.
+
+IPBASE=`/sbin/ifconfig docker0 | grep 'inet addr:'| awk '{print $2}' | sed -e 's/.*://g' | cut -d '.' -f 1-2`
+NEWIP=`docker inspect ideatown_server_1 | grep '"IPAddress' | awk '{sub(/["|,]/, "", $2); print $2}' | sed 's/["|,]//g'`
```

Comment:
That's actually the internal VM IP address, and not useful in the host-side /etc/hosts